action.yml

```diff
@@ -78,6 +78,12 @@ inputs:
   push_code_samples_only:
     description: "This will generate code samples, tag them with the `main` branch name, and push them to the registry. It will not create a pull request or commit any code. This is useful for pushing up some code samples to the registry the first time when there are no code samples in the registry."
     required: false
+  target_directory:
+    description: "The directory the SDK target was generated to"
+    required: false
+  registry_name:
```

Comment:
Why is this being added to outputs

Reply:
I don't see it being populated in outputs anywhere

Reply:
So yeah these were added under inputs and were missing but used